cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,393 @@
+				"\t\t🤨 Reason: " + status.Reason + "\n\n"
+			percentCollectCondition[string(status.Type)+"/"+string(status.Status)] += 1.0 / float64(len(targetsInfo))
+		}
+
+		if i < MAXTARGETSHOW {
+			fmt.Printf(info + "\n\n")
+		}
+	}
+
+	PrintSeparator()
+
+	percentInfo := "Lets look at the overall status...\n"
+	for key, value := range percentCollectCondition {
+		roundedValue := value * 100.00
+		percentInfo += "\tCondition Type/Status: " + key + "\n" +
+			"\tPercent:               " + fmt.Sprint(math.Round(roundedValue*100)/100) + "%\n\n"
+	}
+
+	fmt.Println(percentInfo)
+	PrintSeparator()
+}
+
+func checkKubectl() error {
+	cmd := exec.Command("kubectl", "get", "pods", "-n", "chaos-engineering")
+	_, err := cmd.Output()
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+var contextCmd = &cobra.Command{
+	Use:   "context",
+	Short: "contextualizes disruption config",
+	Long:  `makes use of kubectl to give a better idea of how big the scope of the disruption will be.`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		path, _ := cmd.Flags().GetString("path")
+		return validatePath(path)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		path, _ := cmd.Flags().GetString("path")
+		contextualize(path)
+	},
+}
+
+func contextualize(path string) {
+	var disruption v1beta1.Disruption
+
+	fullPath, err := filepath.Abs(path)
+
+	if err != nil {
+		log.Fatalf("Finding Absolute Path: %v", err)
+	}
+
+	disruptionPath, err := os.Open(filepath.Clean(fullPath))
+
+	if err != nil {
+		log.Fatalf("Openning Yam: %v", err)
+	}
+
+	disruptionBytes, err := ioutil.ReadAll(disruptionPath)
+
+	if err != nil {
+		log.Printf("disruption.Get err   #%v ", err)
+		os.Exit(1)
+	}
+
+	err = goyaml.Unmarshal(disruptionBytes, &disruption)
+
+	if err != nil {
+		log.Fatalf("Unmarshal: %v", err)
+	}
+
+	err = disruption.Spec.Validate()
+
+	if err != nil {
+		log.Fatalf("There were some problems when validating your disruption: %v", err)
+	}
+
+	err = checkKubectl()
+	if err != nil {
+		log.Fatalf("Could not find/use kubectl command, make sure it is in your PATH variable and that all authorizations for the command are set (login to your authorization provider (e.g. AppGate).")
+	}
+
+	targets, err := contextTargetsSize(disruption)
+	if err != nil {
+		log.Fatalf("Could not grab context regarding size and names of pods: %v", err)
+	}
+
+	// Because the state of each of the pods and the state of their containers uses the same information, lets grab
+	// that set of data here and then pass them to other functions to do the searching
+	podsData, nodesData, err := grabDataForTargets(targets, disruption)
+	if err != nil {
+		log.Fatalf("Attempted to grab data for targets and failed: %v", err)
+	}
+
+	if disruption.Spec.Level == POD {
+		printPodStatus(podsData)
+		printContainerStatus(podsData)
+	} else {
+		printNodeStatus(nodesData)
+	}
+
+}
+
+func init() {
+	contextCmd.Flags().String("path", "", "The path to the disruption file to be contextualized.")
```

Comment:
The extended description and/or help text of this command should explain to the user that we're relying on them already having a working `kubectl` connection to a k8s cluster, and that the context we'll be producing is based on their current active `kubectx`